bokehjs/src/lib/models/canvas/canvas.ts

```diff
@@ -96,7 +96,7 @@ export class CanvasView extends UIElementView {
     this.primary = this.create_layer()
     this.overlays = this.create_layer()
     this.overlays_el = div({class: canvas_css.layer})
-    this.events_el = div({class: [canvas_css.layer, canvas_css.events]})
+    this.events_el = div({class: [canvas_css.layer, canvas_css.events], tabIndex: 0})
```

Comment:
Is this needed for anything in this PR?

Reply:
That's the most important part of this PR, otherwise you can't focus a plot.